test/integration/visitor_can_view_cars_test.rb

```diff
@@ -13,22 +13,21 @@ class VisitorCanViewCarsTest < ActionDispatch::IntegrationTest
       assert page.has_content?('Cars')
     end
 
-    assert page.has_content?('1960 Chevy0 El Camino0')
+    assert page.has_content?('1960 El Camino0 Chevy0')
     assert page.has_content?('$100')
-    assert page.has_content?("Dave's Cars")
 
-
-
-    assert page.has_content?('1961 Chevy1 El Camino1')
+    assert page.has_content?('1961 El Camino1 Chevy1')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.